src/adapters/wrappers/model.py

```diff
@@ -167,3 +169,77 @@ def new_init(self, config, *args, **kwargs):
     model = new_model_class.from_pretrained(model_name_or_path, *model_args, **kwargs)
 
     return model
+
+
+def _validate_interface_values(base_model: PreTrainedModel, interface: AdapterModelInterface) -> None:
+    """
+    Validates that all values specified in the interface exist in the model.
+
+    Args:
+        base_model: The base model to validate against
+        interface: The adapter interface to validate
+
+    Raises:
+        ValueError: If any specified path is not found in the model
+    """
+
+    if not multihasattr(base_model, interface.model_embeddings):
+        raise ValueError(
```

Comment:
nit: I think we could use a common template for this error message as it is roughly the same in each instance

Reply:
I thought so too, but then again thought that adding that would introduce slightly more complexity to the code so originally didn't do it. But I've implemented the template now. Thanks for the review!